ShoppingCartExample/Program.cs

```diff
@@ -20,6 +21,17 @@
     siloBuilder.AddMemoryGrainStorage("carts");
 });
 
+builder.Services.AddHostedTemporalWorker(
+        "localhost:7233",
+        "default",
+        CheckoutWorkflow.TaskQueue)
```

Comment:
```suggestion
builder.Services.AddHostedTemporalWorker(CheckoutWorkflow.TaskQueue)
```

Re https://github.com/temporalio/sdk-dotnet/tree/main/src/Temporalio.Extensions.Hosting#worker-services, you don't have to use the client-based overload if you have a client already on the container (which you do next step). (let me know if it doesn't work